src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -427,7 +426,7 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::RestoreDatasetFromNcp(vo
         error = Get(SPINEL_PROP_THREAD_PENDING_DATASET, SPINEL_DATATYPE_VOID_S, &RadioSpinel::ThreadDatasetHandler));
 
 exit:
-    otPlatSettingsDeinit(mInstance);
+    Instance::Get().template Get<SettingsDriver>().Deinit();
```

Comment:
Can we possibly get away with not requiring `template` keyword if we include the `locator-getters.hpp` header in this file?

Reply:
Reading more on this, I don't think it would help since we are already inclining `instance.hpp` header which defines  the template `Get<>` definition.